api/v1alpha1/apidefinition_types.go

```diff
@@ -458,11 +458,10 @@ type APIDefinitionSpec struct {
 	//SessionProvider        SessionProviderMeta `json:"session_provider"`
 	////EventHandlers             EventHandlerMetaConfig `json:"event_handlers"`
 	//EnableBatchRequestSupport bool `json:"enable_batch_request_support"`
-	//EnableIpWhiteListing      bool `json:"enable_ip_whitelisting"`
-	//// +optional
-	//AllowedIPs            []string            `json:"allowed_ips"`
-	//EnableIpBlacklisting  bool                `json:"enable_ip_blacklisting"`
-	//BlacklistedIPs        []string            `json:"blacklisted_ips"`
+	EnableIPWhiteListing bool     `json:"enable_ip_whitelisting,omitempty"`
+	AllowedIPs           []string `json:"allowed_ips,omitempty"`
+	EnableIPBlacklisting bool     `json:"enable_ip_blacklisting,omitempty"`
+	BlacklistedIPs       []string `json:"blacklisted_ips,omitempty"`
```

Comment:
Just some missing doc comments for each of the settings.

See https://github.com/TykTechnologies/tyk-operator/blob/master/api/v1alpha1/apidefinition_types.go#L293-L311 as a pretty good example.